packages/katello/katello/katello.spec

```diff
@@ -194,8 +194,12 @@ Useful utilities for managing Katello services
 %{_sysconfdir}/bash_completion.d/katello-service
 
 %changelog
-* Wed Jul 17 2019 Evgeni Golov - 3.12.1-1
-- Release 3.12.1
+* Sun Jul 21 2019 Jonathon Turel <jturel@gmail.com> 3.13.0-0.4.master
```

Comment:
Changelog doesn't look quite right. I think it should be a single changelog entry for 3.12.1-2 ?